checker/cost_test.go

```diff
@@ -354,6 +354,15 @@ func TestCost(t *testing.T) {
 			hints:  map[string]int64{"str1": 10, "str2": 10},
 			wanted: CostEstimate{Min: 2, Max: 6},
 		},
+		{
+			name: "list size comparison",
+			expr: `list1.size() == list2.size()`,
```

Comment:
Can you expand the test coverage.  E.g. `x > y ? list1.size() : list2.size()`, and some functions that return a scalar other than `size()`?  Are there any other edge cases?

Reply:
Marking this as resolved since coverage on overloaded operators as handled in `functionCost` should now be pretty good (outside of timestamps/durations as per offline discussion on some issues with those - I've created https://github.com/google/cel-go/issues/573 to track resolving that).